rules/otbn.bzl

```diff
@@ -175,3 +214,21 @@ otbn_binary = rv_rule(
     toolchains = ["@rules_cc//cc:toolchain_type"],
     incompatible_use_toolchain_transition = True,
 )
+
+otbn_test = rv_rule(
+    implementation = _otbn_test,
+    test = True,
+    attrs = {
+        "srcs": attr.label_list(allow_files = True),
+        "deps": attr.label_list(providers = [DefaultInfo]),
+        "_cc_toolchain": attr.label(default = Label("@bazel_tools//tools/cpp:current_cc_toolchain")),
+        "_otbn_as": attr.label(default = "//hw/ip/otbn/util:otbn-as", allow_single_file = True),
+        "_otbn_ld": attr.label(default = "//hw/ip/otbn/util:otbn-ld", allow_single_file = True),
```

Comment:
This seems slightly odd to me: we depend on the OTBN toolchain because we need to build stuff. But isn't that implied by the call to `_otbn_binary()` above? I don't know Bazel, but it seems surprising that we'd have to duplicate all of that function's dependencies.

Reply:
Ahh, I see. Thanks for the explanation!